cmd/remco/supervisor.go

```diff
@@ -204,6 +206,7 @@ func (ru *Supervisor) runResource(r []Resource, stop, stopped chan struct{}) {
 			res, err := template.NewResourceFromResourceConfig(ctx, ru.reapLock, rsc)
 			if err != nil {
 				log.Error(err)
+				ru.resourcesWithError++
```

Comment:
I think we need to protect the variable with a mutex or use atomic operations (https://pkg.go.dev/sync/atomic) as we use multiple parallel goroutines here.